Update websocket fork (node-gyp err output on Windows / Node 12)

#### Expected behavior
npm install command is executed without errors

#### Actual behavior
npm install shows errors

#### Steps to reproduce the behavior
1. npm init
2. npm install web 3

#### Logs
> @web3-js/websocket@1.0.30 install C:\Temp\web3.test\node_modules\@web3-js\websocket
> (node-gyp rebuild 2> builderror.log) || (exit 0)


C:\Temp\web3.test\node_modules\@web3-js\websocket>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild ) 
Building the projects in this solution one at a time. To enable parallel build, please add the "-m" switch.
  bufferutil.cc
  win_delay_load_hook.cc
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(34,1): error C2660: 'v8::FunctionTemplate::GetFunction': function does not take 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Users\dummyuser\AppData\Local\node-gyp\Cache\12.14.1\include\node\v8.h(5995,46): message : see declaration of 'v8::FunctionTemplate::GetFunction' (compiling source file ..\src\bufferutil.cc) [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(34,1): error C2661: 'Nan::Set': no overloaded function takes 2 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(50,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(57,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(68,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(70,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(89,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(90,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(92,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(93,1): error C2660: 'v8::Value::Int32Value': function does not take 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Users\dummyuser\AppData\Local\node-gyp\Cache\12.14.1\include\node\v8.h(2613,40): message : see declaration of 'v8::Value::Int32Value' (compiling source file ..\src\bufferutil.cc) [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\bufferutil.cc(94,1): error C2660: 'v8::Value::Int32Value': function does not take 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
C:\Users\dummyuser\AppData\Local\node-gyp\Cache\12.14.1\include\node\v8.h(2613,40): message : see declaration of 'v8::Value::Int32Value' (compiling source file ..\src\bufferutil.cc) [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\bufferutil.vcxproj]
  validation.cc
  win_delay_load_hook.cc
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\validation.cc(114,1): error C2660: 'v8::FunctionTemplate::GetFunction': function does not take 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\validation.vcxproj]
C:\Users\dummyuser\AppData\Local\node-gyp\Cache\12.14.1\include\node\v8.h(5995,46): message : see declaration of 'v8::FunctionTemplate::GetFunction' (compiling source file ..\src\validation.cc) [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\validation.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\validation.cc(114,1): error C2661: 'Nan::Set': no overloaded function takes 2 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\validation.vcxproj]
C:\Temp\web3.test\node_modules\@web3-js\websocket\src\validation.cc(133,1): error C2661: 'v8::Value::ToObject': no overloaded function takes 0 arguments [C:\Temp\web3.test\node_modules\@web3-js\websocket\build\validation.vcxproj]

#### Versions
[NPM, Node, Web3.js, OS, device...]
OS: Win 10, desktop
Node: v12.14.1
Npm: 6.13.4

with node v11.15.0:

C:\Temp\web3.test\node_modules\keccak>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
gyp ERR! configure error
gyp ERR! stack Error: Command failed: C:\Program Files\Python38\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                       ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:299:12)
gyp ERR! stack     at ChildProcess.emit (events.js:193:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:999:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:266:5)
gyp ERR! System Windows_NT 10.0.18363
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Temp\web3.test\node_modules\keccak
gyp ERR! node -v v11.15.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! keccak@2.1.0 rebuild: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the keccak@2.1.0 rebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\dummyuser\AppData\Roaming\npm-cache\_logs\2020-02-11T18_25_14_927Z-debug.log
"Keccak bindings compilation fail. Pure JS implementation will be used."

> secp256k1@3.8.0 install C:\Temp\web3.test\node_modules\secp256k1
> npm run rebuild || echo "Secp256k1 bindings compilation fail. Pure JS implementation will be used."

--- a/packages/web3-providers-ws/package.json
+++ b/packages/web3-providers-ws/package.json
@@ -13,10 +13,10 @@
   "types": "types/index.d.ts",
   "main": "src/index.js",
   "dependencies": {
-    "@web3-js/websocket": "^1.0.29",
     "eventemitter3": "^4.0.0",
     "underscore": "1.9.1",
-    "web3-core-helpers": "1.2.8"
+    "web3-core-helpers": "1.2.8",
+    "websocket": "^1.0.31"
   },
   "devDependencies": {
     "dtslint": "^3.4.1",

--- a/packages/web3-providers-ws/src/index.js
+++ b/packages/web3-providers-ws/src/index.js
@@ -25,7 +25,7 @@
 var EventEmitter = require('eventemitter3');
 var helpers = require('./helpers.js');
 var errors = require('web3-core-helpers').errors;
-var Ws = require('@web3-js/websocket').w3cwebsocket;
+var Ws = require('websocket').w3cwebsocket;
 
 /**
  * @param {string} url

